<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <title>Winter Seminar Series</title>

    <!-- css -->
    <link rel="stylesheet" href="../bower_components/bootstrap/dist/css/bootstrap.min.css">
    <link rel="stylesheet" href="../bower_components/ionicons/css/ionicons.min.css">
    <link rel="stylesheet" href="../assets/css/main.css">
</head>
<body data-spy="scroll" data-target="#site-nav">
    <nav id="site-nav" class="navbar navbar-fixed-top navbar-custom">
        <div class="container">
            <div class="navbar-header">

                <!-- logo -->
                <div class="site-branding">
                    <a class="logo" href="../index.html">
                        
                        <!-- logo image  -->
                        <img src="../assets/images/wss_logo.png" alt="Logo" width="45px">

                        Winter Seminar Series
                    </a>
                </div>

                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-items" aria-expanded="false">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>

            </div><!-- /.navbar-header -->

            <div class="collapse navbar-collapse" id="navbar-items">
                <ul class="nav navbar-nav navbar-right">

                    <!-- navigation menu -->
                    <!-- <li class="active"><a data-scroll href="#about">About</a></li> -->
                    <!-- <li><a data-scroll href="#speakers">Speakers</a></li> -->
                    <!-- <li><a data-scroll href="#partner">Sponsors</a></li> -->
                    <!-- <li><a data-scroll href="#schedule">Schedule</a></li> -->     
                    <!-- <li><a data-scroll href="#">Sponsorship</a></li> -->
                    <!-- <li><a data-scroll href="#faq">FAQ</a></li> -->
                    <!-- <li><a data-scroll href="#photos">Photos</a></li> -->
                
                </ul>
            </div>
        </div><!-- /.container -->
    </nav>

    <header id="site-header" class="site-header valign-center"> 
        <div class="intro">

            <h1>Winter Seminar Series</h1>
            <h1>Advanced Topics in Computer Science and Engineering</h1>

            <h2>Sharif University of Technology / 28-29 December 2016</h2>
            
            <!-- <p>First &amp; Largest Conference</p> -->
            
            <a class="btn btn-white" data-scroll href="#about">Register Now</a>
        
        </div>
    </header>

    <section id="single-speaker" class="section single-speaker">
        <div class="container">
			
			<div class="row">
				<div class='col-md-4'><div class='speaker'><figure><img alt='' class='img-responsive center-block' src='../assets/images/speakers/ehsan-asgari.jpg' width='250px'></figure><h4>Ehsan Asgari</h4><p>PhD Candidate, University of California, Berkeley</p></div></div>
				<div class='col-md-8'><div class='single-speaker'>
				<h4>Title</h4>
				<p>Continuous Distributed Representation of Biological Sequences and Their Applications in Bioinformatics</p>
				<h4>Abstract</h4>
				<p>Biophysical and biochemical principles govern biological sequences (e.g., DNA, RNA, and protein sequences) similar to the way grammar of a natural language determines the structure of clauses and sentences. This analogy motivates ‘life language processing’, i.e. treating biological sequences as the output of a certain language and adopt/develop language processing methods to preform analyses and predictions in that language. We propose two specific aims for life language processing: (1) Developing computational linguistics representation learning for biological sequences: the large gap between the number of known sequences (raw data) versus the number of known functions/structures associated with these sequences (meta-data), encourage us to develop methods that can obtain prior knowledge from the existing sequences. Continuous vector representations of words known as word vectors have recently become popular in natural language processing (NLP) as an efficient unsupervised approach to represent semantic/syntactic units of text helping in the downstream NLP tasks (e.g., machine translation, part-of-speech tagging, information retrieval, etc.). In this work, we propose distributed vector representations of biological sequence segments (n-grams), called bio-vectors, using skip-gram neural network. We propose intrinsic evaluation of bio-vectors by measuring the continuity of the underlying biophysical and biochemical properties (e.g., average mass, hydrophobicity, charge, and etc.). In addition to intrinsic evaluations, for the purpose of extrinsic evaluations, we have employed this representation in classification of 324018 protein sequences belonging to 7027 protein families, where an average family classification accuracy of 93%±0.06% was obtained. In addition, incorporation of bio-vector representation versus one-hot vector features in Maxmargin Markov Network (M3Net) for intron-exon prediction and domain identification tasks could improve the sequence labeling accuracy from 73.84% to 74.99% and from 82.4% to 89.8%, respectively. (2) Performing computational linguistics comparison of genomic language variations: the purpose of this aim is to quantify the distances between syntactic and semantic features of two genomic language variations, with various applications in comparative genomics. Training model of bio-vectors is analogous to neural probabilistic language modeling. Hence, such representations can characterize sequences in terms of underlying biochemical and biophysical patterns. This makes the network of n-grams in this space an indirect representation of the underlying language model. Considering this fact, we propose a new quantitative measure of distance between genomic language variations based on the divergence between networks of ngrams in different genetic variations, called word embedding language divergence. We perform language comparison for the coding regions in the genomes of 12 different organisms (4 plants, 6 animals, and two human subjects). Our results confirm a significant high-level difference in the genetic language model of humans/animals versus plants. The proposed method is a step toward defining a new quantitative measure of similarity between genomic languages, with applications in characterization/classification of sequences of interest.</p>
				<h4>Bio</h4>
				<p>Ehsaneddin Asgari is a PhD candidate at University of California, Berkeley. His PhD research explores the development of deep learning, specifically deep language processing methods for performing analysis and predictions on genomics and metagenomics sequences. His research interests are in the areas of Bioinformatics and Natural Language Processing. He is a former researcher at Deep Language Processing group at LMU, Genesis group at MIT Computer Science and Artificial Intelligence Laboratory, Neuroscience Statistics Lab at MIT Brain and Cognitive Science Department, ABB Corporate Research, Audiovisual Communications Lab at EPFL, ADSC of UIUC, and Digital Media Lab. He received his M.Sc. degrees from UC Berkeley and EPFL and his B.Sc. from Sharif University of Technology in Computer Engineering.</p>
				</div></div>
			</div>
			
		</div>
	</section>

    <footer class="site-footer">
        <div class="container">
            <div class="row">
                <div class="col-md-12">
                    <p class="site-info">Designed and <br> Developed by <a href="http://technextit.com">Technext Limited</a></p>
                    <!-- <ul class="social-block">
                        <li><a href=""><i class="ion-social-twitter"></i></a></li>
                        <li><a href=""><i class="ion-social-facebook"></i></a></li>
                        <li><a href=""><i class="ion-social-linkedin-outline"></i></a></li>
                        <li><a href=""><i class="ion-social-googleplus"></i></a></li>
                    </ul> -->
                </div>
            </div>
        </div>
    </footer>

    <!-- script -->
    <script src="../bower_components/jquery/dist/jquery.min.js"></script>
    <script src="../bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
    <script src="../bower_components/smooth-scroll/dist/js/smooth-scroll.min.js"></script>
    <script src="../assets/js/main.js"></script>
</body>
</html>